modules/page-memory/src/test/java/org/apache/ignite/internal/pagememory/persistence/store/AbstractFilePageStoreIoTest.java

```diff
@@ -405,6 +404,6 @@ void testRenameAndReadRace(FileIoFactory ioFactory) throws Exception {
     }
 
     private static FileIoFactory[] ioFactories() {
-        return new FileIoFactory[]{new RandomAccessFileIoFactory(), new AsyncFileIoFactory()};
+        return new FileIoFactory[]{new RandomAccessFileIoFactory()};
```

Comment:
I think it's worth returning to the way it was, because we can return to a different implementation.